hack/test-go.sh

```diff
@@ -135,33 +136,50 @@ if [[ $iterations -gt 1 ]]; then
   fi
 fi
 
+cover_report_dir=""
+combined_cover_profile=""
+if [[ -n "${KUBE_COVER}" ]]; then
+  cover_report_dir="/tmp/k8s_coverage/$(kube::util::sortable_date)"
+  combined_cover_profile="${cover_report_dir}/combined-coverage.out"
+  kube::log::status "Saving coverage output in '${cover_report_dir}'"
+  mkdir -p ${cover_report_dir}
+  echo "mode: ${KUBE_COVERMODE}" >${combined_cover_profile}
```

Comment:
Can you comment what's going on with the `mode:`? I worked it out but it would be kind to the next reader.